docs/examples/example.ipynb

```diff
@@ -327,58 +303,54 @@
     "\n",
     "```python\n",
-    "# modular-arm/src/my_modular_arm.py\n",
+    "# wifi-sensor/src/wifi_sensor.py\n",
     "import asyncio\n",
-    "from typing import Any, ClassVar, Dict, Mapping, Optional, Tuple\n",
+    "from typing import Any, ClassVar, Dict, Mapping, Optional\n",
     "from typing_extensions import Self\n",
     "\n",
-    "from viam.components.arm import Arm, JointPositions, KinematicsFileFormat, Pose\n",
+    "from viam.components.sensor import Sensor\n",
     "from viam.operations import run_with_operation\n",
     "from viam.proto.app.robot import ComponentConfig\n",
     "from viam.proto.common import ResourceName\n",
     "from viam.resource.base import ResourceBase\n",
     "from viam.resource.types import Model, ModelFamily\n",
     "\n",
+    "class MySensor(Sensor):\n",
+    "    # Subclass the Viam Sensor component and implement the required functions\n",
+    "    MODEL: ClassVar[Model] = Model(ModelFamily(\"acme\",\"wifi_sensor\"), \"linux\")\n",
     "\n",
-    "class MyModularArm(Arm):\n",
-    "    # Subclass the Viam Arm component and implement the required functions\n",
-    "    MODEL: ClassVar[Model] = Model(ModelFamily(\"acme\", \"demo\"), \"myarm\")\n",
-    "    \n",
-    "    def __init__(self, name: str):\n",
-    "        ...\n",
-    "    \n",
     "    @classmethod\n",
     "    def new(cls, config: ComponentConfig, dependencies: Mapping[ResourceName, ResourceBase]) -> Self:\n",
-    "        arm = cls(config.name)\n",
-    "        return arm\n",
+    "        sensor = cls(config.name)\n",
+    "        return sensor\n",
     "```\n",
     "\n",
     "After the resource has a defined model and creator function, the resource must be registered to the `Registry`. This can be done using [`register_resource_creator()`](https://python.viam.dev/autoapi/viam/resource/registry/index.html#viam.resource.registry.Registry.register_resource_creator) and passing a [`ResourceCreatorRegistration`](https://python.viam.dev/autoapi/viam/resource/registry/index.html#viam.resource.registry.ResourceCreatorRegistration) object with the creator function as a parameter.\n",
     "\n",
-    "In the same `modular-arm/src` directory, create a new file called `__init__.py`. In it, call `Registry.register_resource_creator()` with the subtype of the resource that was subclassed (in this case, `Arm`), the model name, and a `ResourceCreatorRegistration` object with the `new` function defined in the modular resource.\n",
+    "In the same `src` directory, create a new file called `__init__.py`. In it, call `Registry.register_resource_creator()` with the subtype of the resource that was subclassed, the model name, and a `ResourceCreatorRegistration` object with the creator function defined in the modular resource.\n",
     "\n",
     "\n",
     "```python\n",
```

Comment:
Should I copy this code into my `__init__.py` file? If so, please introduce the code with this guidance. Perhaps:

```
"Copy the code below into your `__init__.py` file:\n",
```